molecule/check_facts/tasks/check.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- fail:
+    msg: "Task 'check_task' wasn't started"
+  when: delivered_package_path_from_check_task is not defined
+  run_once: true
+
+- fail:
+    msg: "Incorrect value of 'delivered_package_path', got: '{{ delivered_package_path_from_check_task }}'"
+  when: delivered_package_path_from_check_task != "my_task_default"
```

Comment:
`delivered_package_path != "my_task_default"`

Reply:
Oh no. I can't understand what does this test do...

Reply:
It's test that variable delivered_package_path is empty